### DISCUSSION TITLE
Reporting & Updating

### DISCUSSION THREAD

**Discussion by ATLCOTech** [code4recovery/12-step-meeting-list#1494] [Q&A]
Hi Team, 

I wanted to ask a question. In the meeting guide app there a Meeting called Franklin Twelve Steppers Group. When you scroll down it says that Central Office of Dayton is the POC for updating. However, when you click on contact>Update Meeting Information it shows Cincinnati Intergroup, Central Office of Dayton, Area 56 District 12 and 13 and Area 26 Kentucky.

1. It's not on the Cincinnati website at all
2. should it be showing all of those POC's as the updating entity?

This is the case with many of the meetings in the Tri-State area of of Norther Kentucky, SW Ohio and SE Indiana.

**Comment by joshreisner** [comment]
@ATLCOTech when you go to Contact > Update Meeting Information, you are seeing all the entities providing meeting information to you. 

that particular meeting is provided by the Central Office of Dayton, but the other entites are providing other meetings. there are usually 500 meetings on your device at any one time

**Comment by ATLCOTech** [comment]
Ok. It’s causing a great deal of confusing for those digging into the app
as to why the other entities are present. We have a meeting tonight so I’ll
just let them know the below.

So the entity listed on the main page is the correct one and the owner of
the information?

Sincerely,

Mark J
Tech & Newsletter Chair

If you have a newsletter submission, please send to ***@***.***
.
If you have a website comment, issue or would like to join the tech/web
committee email ***@***.*** ***@***.***>
To contact the steering committee email ***@***.***
Central Office ***@***.***

**Reply by joshreisner** [reply]
both screens are correct 👍 

each meeting screen shows which entity supplied that particular listing

the `contact > update information` screen shows all the entities supplying information across all days of the week

if that screen weren't there, people would reach out to GSO to request updates - that screen is an important signal to users that they are receiving the listings from local service entities

**Comment by ATLCOTech** [comment]
So that meeting isn’t listed on Cincinnati’s page at all. I have access to
the back end and it’s not there.

Sincerely,

Mark J
Tech & Newsletter Chair

If you have a newsletter submission, please send to ***@***.***
.
If you have a website comment, issue or would like to join the tech/web
committee email ***@***.*** ***@***.***>
To contact the steering committee email ***@***.***
Central Office ***@***.***

**Reply by joshreisner** [reply]
ok, not sure why that would be surprising. the meeting listing says it comes from the dayton central office. some entities list some meetings, other entities list others. presumably cincinnati lists meetings closer to cincinnati, and dayton closer to dayton

whenever you open the app you are going to see a mix of meetings from various offices. the contact tab shows you contact information from all the offices

**Comment by ATLCOTech** [comment]
Ugh the file was too large. Here's the link to my dropbox.

https://www.dropbox.com/scl/fi/4drumydzg5jsxf3asojhs/RPReplay_Final1724362577.MP4?rlkey=40glfvhezc6je5smvo76rckki&dl=0

Sincerely,

Mark J
Tech & Newsletter Chair

If you have a newsletter submission, please send to ***@***.***
.
If you have a website comment, issue or would like to join the tech/web
committee email ***@***.*** ***@***.***>
To contact the steering committee email ***@***.***
Central Office ***@***.***

**Reply by joshreisner** [reply]
yes that's how it's supposed to be 👍

**Comment by ATLCOTech** [comment]
Ok. Thank you soo much for your patience.
Sincerely,

Mark J
Tech & Newsletter Chair

If you have a newsletter submission, please send to ***@***.***
.
If you have a website comment, issue or would like to join the tech/web
committee email ***@***.*** ***@***.***>
To contact the steering committee email ***@***.***
Central Office ***@***.***
